manifests/install.pp

```diff
@@ -100,7 +100,7 @@
               architecture => 'amd64,arm64,armhf',
               repos        => 'main',
               key          => {
-                'id'     => $grafana::repo_key_id,
+                'name'   => 'grafana.asc',
```

Comment:
@saz correct me if i'm wrong, but if we change it like that, there should be https://apt.grafana.com/grafana.asc.

for me it's 404, but https://apt.grafana.com/gpg.key is working

Reply:
`$grafana::repo_gpg_key_url` is set to `https://packages.grafana.com/gpg.key` by default

Reply:
sorry, my bad. i misread something